subprojects/dependency-management/src/main/java/org/gradle/api/internal/artifacts/transform/DefaultVariantTransformRegistry.java

```diff
@@ -99,7 +99,7 @@ public void registerTransform(Action<? super VariantTransform> registrationActio
             ArtifactTransformRegistration finalizedRegistration = registrationFactory.create(registration.from.asImmutable(), registration.to.asImmutable(), actionType, parameterObject);
             transforms.add(finalizedRegistration);
         } catch (Exception e) {
-            throw new VariantTransformConfigurationException(String.format("Cannot register artifact transform %s with parameters %s", ModelType.of(actionType).getDisplayName(), parameterObject), e);
+            throw new VariantTransformConfigurationException(String.format("Cannot register artifact transform %s (from: %s, to: %s)", ModelType.of(actionType).getDisplayName(), registration.from, registration.to), e);
```

Comment:
Nit:

```suggestion
            throw new VariantTransformConfigurationException(String.format("Cannot register artifact transform %s (from %s to %s)", ModelType.of(actionType).getDisplayName(), registration.from, registration.to), e);
```

...since the actual values are wrapped in `{...}` anyway.

Reply:
Done.